src/prices/PricesService.ts

```diff
@@ -115,12 +127,16 @@ export default class PricesService {
     )
   }
 
-  private async cUSDToLocalCurrency(
+  private async USDToLocalCurrency(
```

Comment:
nit: functions should use camelCase
```suggestion
  private async usdToLocalCurrency(
```